storage/innobase/include/log0log.h

```diff
@@ -380,4 +380,20 @@ void log_crash_safe_validate(log_t &log);
 
 #endif /* !UNIV_HOTBACKUP */
 
+#ifdef XTRABACKUP
```

Comment:
if this code is moved around, this shouldn't be under ifdef XTRABACKUP tag.

Reply:
fixed.